modules/ts/src/ts_func.cpp

```diff
@@ -1063,6 +1067,96 @@ static void setpos( const Mat& mtx, vector<int>& pos, size_t idx )
     }
 }
 
+template <typename T>
+struct reduceMinMaxImpl
```

Comment:
Prefer to move that code into `core/test`
Lets keep `ts` modules small

Reply:
Are you proposing to move refop implementation in `test_arithm.cpp`? It seems to contain tests only. Or is it better to create `core/test/reduce_arg_refop.hpp` + `core/test/reduce_arg_refo.cpp`?

Reply:
I believe single `ref_reducearg.impl.hpp` file with static functions should be enough (and include it into test_arithm.cpp where it is used).
Prefer to do NOT create additional .cpp file.